### PR TITLE
Add configuration for ssl_verify_hostname

### PR DESCRIPTION
**Pull request by ogibayashi**
Hi. I would like to configure `ssl_verify_hostname` of ruby-kafka which is added in https://github.com/zendesk/ruby-kafka/pull/733

### CODE REVIEW

**Comment by repeatedly** [comment]
Thanks. I will check after back from kubecon.

File: fluent-plugin-kafka.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "fluentd", [">= 0.10.58", "< 2"]
   gem.add_dependency 'ltsv'
-  gem.add_dependency 'ruby-kafka', '>= 0.7.1', '< 0.8.0'
+  gem.add_dependency 'ruby-kafka', '>= 0.7.8', '< 0.8.0'
```
Comment (ogibayashi) [inline fluent-plugin-kafka.gemspec]: `ssl_verify_hostname` is available with ruby-kafka >= 0.7.8.

**Comment by ganmacs** [comment]
Doesn't it need to add the same change as out_kafka2.rb to out_kafka.rb?

**Comment by ogibayashi** [comment]
That's right. I have added the change to out_kafka2.rb and out_kafka_buffered.rb.

**Comment by ganmacs** [comment]
I noticed now but are in_kafka.rb and in_kafka_group.rb also needed?

**Comment by repeatedly** [comment]
I think adding it better but not required for this patch.

**Comment by repeatedly** [comment]
Thanks!

**Comment by ogibayashi** [comment]
Thank you!

**Pull request merged**